docs/content/preview/faq/yugabyte-platform.md

```diff
@@ -123,3 +138,101 @@ Yes, you can control what YugabyteDB Anywhere is spinning up. For example:
+- Invoke procedures or methods on the node agent.
+- Additionally, for on premises (manually provisioned nodes) deployments, node agent also functions as a utility to run preflight checks, and add node instances.
+
+### How is node agent installed on a YugabyteDB node?
+
+Installation depends on how the nodes are provisioned.
+
+For cloud (AWS, GCP, and Azure) and non-manual on-premises (sudo access provided) providers, agents are automatically installed on each universe node during provisioning using SSH. A sudo user with SSH access (for example, ec2-user for AWS) is required. After the node agent is installed, all the legacy SSH calls are replaced with node agent calls. Just like an SSH daemon, node agent is run as a root user to perform provisioning of the nodes. After node provisioning, you can revoke the SSH access, but it's recommended to enable the access for debugging.
+
+For manually provisioned on-premises providers, node agent is installed on each node either using a script or manually as part of the manual provisioning process. For more information, refer to [Manually provision on-premises nodes](../../yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual/).
+
+### Why does YBA prompt for SSH details if the node agent is installed manually and can replace SSH?
+
+When creating an on-premises provider, you are prompted to provide SSH credentials. After you have provisioned and added the instances to the provider (including installing the node agent), YugabyteDB Anywhere no longer requires SSH or sudo access to nodes.
+
+If you are manually provisioning nodes, these credentials aren't needed and you can provide a fictitious non-working SSH key.
+
+### What is registration and unregistration of node agent?
+
+Node agent is a secure service that authenticates every remote call from YBA. This requires the following:
+
+- Making YBA aware of the node agent service.
+- Establishing trust between the node agent and YBA.
+
+This process of establishing a secure connection is called registration. Note that registration has nothing to do with a provider.
+
+Unregistration is the process of removing the node agent entry from YBA such that there is no further communication.
+
+### Why does node agent installation ask for provider and other details during on-prem manual node agent setup?
+
+Node agent registration internally does not need configuration values such as region, availability zone, and so on, but it still requests the details because of the additional preflight check capability that it provides. It stores the configuration details in a config file called `$NODE_AGENT_HOME/config/config.yml`, and the values are used when the preflight check is run, or the node is added to the node instance of the provider. This is convenient when you run the command as you don't have to provide the input values again.
```

Comment:
As mentioned above, node agent is used to run pre-flight checks on the node during various day-0 and day-2 operations. These pre-checks need information like the non-root user's home directory, expected port number for Node exporter, the NTP servers etc. which are attributes configured with a provider. In order to run these pre-flight checks the node agent needs to be configured to a provider in YBA. These details are needed to make the node agent aware of the YBA provider which the node will become a part of.